charts/lagoon-remote/templates/ssh-core.clusterrole.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.sshCore.enabled -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "lagoon-remote.sshCore.fullname" . }}
+  labels:
+    {{- include "lagoon-remote.sshCore.labels" . | nindent 4 }}
+rules:
+# k8s.ensureScaled
```

Comment:
Sorry it isn't very clear but these comments in the ssh-portal ClusterRole align with functions in the ssh-portal codebase to explain where the permissions are used. e.g. https://github.com/uselagoon/lagoon-ssh-portal/blob/main/internal/k8s/exec.go#L117

So maybe they could be  updated to explain which part of ssh-core uses each permission? Or just removed if you don't think it's helpful.